.changeset/silent-birds-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@igloo-ui/tooltip': minor
```

Comment:
Changing the default position implies a major versioN. A description must also be added to the BREAKING CHANGE